ligo/src/registryDAO.mligo

```diff
@@ -194,3 +233,13 @@ let default_registry_DAO_full_storage (admin, token_address, a, b, s_max, c, d
     decision_lambda = registry_DAO_decision_lambda;
     } in
   (startup, (new_storage, new_config))
+
+let successful_proposal_receiver_view (full_storage : full_storage): proposal_receivers =
```

Comment:
Can you please add a brief comment here, just to point out that this isn't actually used (I'm not sure it will ever be, but for now let's keep it).